CondTools/Geometry/test/me0geometrywriter.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
@jshlee - this one suppose to test ME0 only. The test for GEM is this one: https://cmssdt.cern.ch/lxr/source/CondTools/Geometry/test/gemgeometrywriter.py, but the actual DB writers are in test/writehelpers directory. I suggest, you revert modifications of this file, e.g. it is not needed in this PR. Thanks.